src/verify.rs

```diff
@@ -1,26 +1,9 @@
 use bit_set::Bitset;
-use diag::Diagnostic;
-use nameck::Atom;
-use nameck::Nameset;
-use parser;
-use parser::Comparer;
-use parser::copy_token;
-use parser::NO_STATEMENT;
-use parser::Segment;
-use parser::SegmentId;
-use parser::SegmentOrder;
-use parser::SegmentRef;
-use parser::StatementAddress;
-use parser::StatementRef;
-use parser::StatementType;
-use parser::TokenPtr;
-use scopeck::ExprFragment;
-use scopeck::Frame;
-use scopeck::Hyp;
-use scopeck::ScopeReader;
-use scopeck::ScopeResult;
-use scopeck::ScopeUsage;
-use scopeck::VerifyExpr;
+use diag::{Diagnostic, Result};
```

Comment:
How do you sort imports automatically? rustfmt isn't doing anything for me in this regard. How about just keeping imports unsorted / whatever strikes your fancy, until rustfmt starts doing it automatically (at which point that becomes standard)? Also, are you using a non-default rustfmt.toml?